nucleus/__init__.py

```diff
@@ -131,13 +131,18 @@ def __init__(self, api_key: str, use_notebook: bool = False):
         if use_notebook:
             self.tqdm_bar = tqdm.notebook.tqdm
 
-    def list_models(self) -> List[str]:
+    def list_models(self) -> List[Model]:
         """
         Lists available models in your repo.
         :return: model_ids
         """
-        # TODO implement API
-        raise NotImplementedError
+        model_objects = self._make_request({}, "models/", requests.get)
+
+        try:
+            models = [Model(model['id'], model["name"], model["ref_id"], model["metadata"], self) for model in model_objects['models']]
+            return models
+        except e:
```

Comment:
Why do we have a blanket try/except here?

Reply:
catching errors if json is badly formatted/ no models exist returns empty array

Reply:
If no models exist (i.e. `model_objects['models']` is an empty array), then the list comprehension will be an empty array, it won't crash, so this is unnecessary. 

If the JSON for one model is badly formatted, we should not silently repress all results for all models. The error should be loud and noisy—this means there is a server-side bug on our end.

In general, there is almost never a good reason to have a blanket except statement that catches every possible type of exception. Should avoid this antipattern.